src/app/pages/fund/components/fast-checkout-badge.tsx

```diff
@@ -20,7 +20,7 @@ export function FastCheckoutBadge() {
       spacing="extra-tight"
     >
       <FiZap color="#008051" size="12px" strokeWidth="2.5px" />
-      <Caption color="#008051" fontWeight={500} variant="c2">
+      <Caption color="#008051" fontWeight={500}>
```

Comment:
@mica000 @fabric-8 : Can you please help us update this to our new colours? For the `Fast Checkout`, what colour should we use for `#008051`

![Screenshot 2023-11-09 at 09 52 20](https://github.com/leather-wallet/extension/assets/2938440/5412ace1-e631-4345-9c38-59ca013a7514)